tests/test_cli.py

```diff
@@ -99,6 +99,10 @@ def test_add_iov(datadir, options, tmp_path):
         assert re.search(r'ETAI1', mod_cov)
 
 
+@pytest.mark.filterwarnings(
```

Comment:
Should the filtering be done in the actual code triggering it or would that be sweeping it under the carpet for the user?

Reply:
I think you are correct that it would be better user-wise to filter it in the actual code.

Reply:
Changed!